styles.css

```diff
@@ -127,3 +132,172 @@ body {
+  }
+
+  .logo__circle_color_blue,
+  .logo__circle_color_dark-blue {
+    margin-left: 53px;
+  }
+
+  .menu {
+    padding-top: 0;
+    padding-right: 39px;
+  }
+
+  .menu__item {
+    font: 500 18px/21px Roboto, arial, helvetica, sans-serif;
+  }
+
+  .header {
+    margin-bottom: 10%;
+  }
+
+  .main {
+    padding: 38px 50% 0 50px;
+    background: top -30px right 6% /47% no-repeat url(img/img.svg);
+  }
+
+  .main__title {
+    font: bold 66px/88px 'pt_serif', arial, helvetica, sans-serif;
+    margin-bottom: 35px;
+  }
+
+  .main__description {
+    max-width: 60%;
+    font: 22px/26px Roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    font: 500 18px/21px Roboto, arial, helvetica, sans-serif;
+    padding: 17px 28px;
+    margin-left: 2px;
+    margin-bottom: 25%;
+  }
+}
+
+@media (max-width: 768px) {
+  #checkbox {
+    opacity: 0;
+  }
+
+  .burger__menu {
+    display: flex;
+    width: 25px;
+    padding-top: 4px;
+    border-bottom: 3px solid #0056ec;
+  }
+
+  .burger {
+    display: block;
+    position: relative;
+    margin: 27px 8px 0 0;
+    z-index: 20;
+  }
+
+  .burger::after, .burger::before {
+    display: flex;
+    position: relative;
+    content: "";
+    border-bottom: 3px solid #0056ec;
+    padding-bottom: 4px;
+  }
+
+  #checkbox:checked ~ label .burger::before {
+    position: relative;
+    transform: rotate(45deg);
+    top: 7px;
+  }
+
+  #checkbox:checked ~ label .burger::after {
+    position: relative;
+    transform: rotate(-45deg);
+    top: -4px;
+    right: 3px;
+  }
+
+  #checkbox:checked ~ label .burger__menu {
+    border: none;
+  }
+
+  .menu {
+    display: none;
+  }
+  
+  #checkbox:checked ~ .menu {
+    display: block;
+    position: fixed;
+    background: white;
+    padding-top: 50%;
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    z-index: 10;
+  }
+
+  #checkbox:checked ~ .menu .menu__item {
+    display: block;
+    font: 500 30px/35px Roboto, arial, helvetica, sans-serif;
+    padding-bottom: 24px;
+    width: auto;
+    margin-right: 0;
+  }
+
+  .wrapper {
+    width: 100%;
+    margin: auto;
+  }
+
+  .logo {
+    padding-top: 35px;
+  }
+
+  .logo__text {
+    display: none;
+  }
+
+  .logo__circle_color_dark-blue {
+    margin-left: 23px;
+  }
+
+  .logo__circle_color_blue {
+    margin-left: 23px;
+  }
+
+  .header {
+    margin-bottom: auto;
+  }
+
+  .main {
+    text-align: center;
+    padding: 70% 0 0 0;
+    background-size: 80%;
+    background-position: top 5px right 26px;
+  }
+
+  .main__title {
+    font: bold 30px/40px 'pt_serif', arial, helvetica, sans-serif;
+    padding-top: 17px;
+    margin-left: 14px;
+    margin-bottom: 32px;
+  }
+
+  .main__description {
+    margin-bottom: 25px;
+    max-width: 68%;
+    font: 16px/19px Roboto, arial, helvetica, sans-serif;
+    margin-left: 19%;
+  }
+
+  .main__button {
+    font: 500 14px/16px Roboto, arial, helvetica, sans-serif;
+    margin-left: 5%;
+    }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6278784)